packages/machines/select/src/select.connect.ts

```diff
@@ -205,6 +205,10 @@ export function connect<T extends PropTypes, V extends CollectionItem = Collecti
       dir: state.context.dir,
       "aria-hidden": true,
       "data-state": isOpen ? "open" : "closed",
+      "data-disabled": dataAttr(isDisabled),
+      "data-invalid": dataAttr(isInvalid),
+      "aria-invalid": isInvalid,
```

Comment:
We don't need `aria-invalid` here

```suggestion
      "aria-invalid": isInvalid,
```

Reply:
Not a deal breaker, but what if we want to change color of indicator i case a select field is invalid?

Or is this some semantic conflict?